From 5f28f1adf6a48887958bdacfcd1e70bb185a9088 Mon Sep 17 00:00:00 2001
From: Alfred Neumayer <dev.beidl@gmail.com>
Date: Wed, 21 Oct 2020 17:54:50 +0000
Subject: [PATCH] (halium) don't fail with error on missing HYBRIS_DATA_PART

halium-boot does not use fixup-mountpoints, so disable it
until hybris-boot will become relevant again.

Change-Id: I9a3663d200e9b1c1052ad270b481fc13986afbe8
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 0c14b59..2e56279 100644
--- a/Android.mk
+++ b/Android.mk
@@ -72,7 +72,7 @@ $(warning ********************* /boot appears to live on $(HYBRIS_BOOT_PART))
 $(warning ********************* /data appears to live on $(HYBRIS_DATA_PART))
 
 ifneq ($(words $(HYBRIS_DATA_PART)),1)
-$(error There should be a one and only one device entry for HYBRIS_DATA_PART)
+#$(error There should be a one and only one device entry for HYBRIS_DATA_PART)
 endif
 
 BB_STATIC := $(PRODUCT_OUT)/utilities/busybox
-- 
2.17.1

